lib/main.js

```diff
@@ -261,7 +275,7 @@ class MyAlgoConnect {
 
 			const res = await this.bridge.sendMessage(
 				this.currentConnectPopup,
-				{ method: "unlock", params: { shouldSelectOneAccount: settings.shouldSelectOneAccount } },
+				{ method: "unlock", params: Object.assign(settings, { disableLedgerNano: this.disableLedgerNano }) },
```

Comment:
We need to start to use _.deepCopy fron loadash library instead  this function, this is a shallow copy

Reply:
We don't need a deep copy, only merging both objects.